services/src/main/java/org/keycloak/protocol/saml/mappers/ScriptBasedMapper.java

```diff
@@ -129,19 +130,17 @@ public void transformAttributeStatement(AttributeStatementType attributeStatemen
                 bindings.put("userSession", userSession);
                 bindings.put("keycloakSession", session);
             });
-            //If the result is a an array or is iterable, get all values
-            if (attributeValue.getClass().isArray()){
```

Comment:
This way of checking for an array does not work. Instead we want to use the `ScriptObjectMirror#isArray` method.